### ISSUE TITLE
Failing to install zfsutils-linux and debootstrap

### ISSUE THREAD

**Issue by RNCTX** [jen20/packer-ubuntu-zfs#5]
Hey there, thanks for the scripts.

I struggled to get them to work initially, it seems that there are issues with the order in which required items are installed.

```
amazon-ebssurrogate: + apt-get install -y zfsutils-linux debootstrap gdisk

    amazon-ebssurrogate: Reading package lists... Done
    amazon-ebssurrogate: Building dependency tree
    amazon-ebssurrogate: Reading state information... Done
    amazon-ebssurrogate: Package debootstrap is not available, but is referred to by another package.
    amazon-ebssurrogate: This may mean that the package is missing, has been obsoleted, or
    amazon-ebssurrogate: is only available from another source
    amazon-ebssurrogate:
    amazon-ebssurrogate: E: Unable to locate package zfsutils-linux
    amazon-ebssurrogate: E: Package 'debootstrap' has no installation candidate
==> amazon-ebssurrogate: Terminating the source AWS instance...
==> amazon-ebssurrogate: Deleting temporary security group...
==> amazon-ebssurrogate: Deleting temporary keypair...
Build 'amazon-ebssurrogate' errored: Script exited with non-zero exit status: 100

==> Some builds didn't complete successfully and had errors:
--> amazon-ebssurrogate: Script exited with non-zero exit status: 100

==> Builds finished but no artifacts were created.

```

Changing the order to gdisk -> zfsutils-linux -> debootstrap seems to be working, though.

**Comment by jen20** [comment]
Thanks, this should have been addressed by #6.

**Comment by jen20** [comment]
In retrospect, I don't think the resolution here was quite correct. I've fixed this in the new 20.04 builder here:

https://github.com/jen20/packer-ubuntu-zfs/blob/master/focal/scripts/surrogate-bootstrap.sh#L9-L13

I'll back port this to the xenial and bionic builders also. Thanks for reporting @RNCTX!
